### PR TITLE
Misc docs updates

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Docs

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/interop/wasm-javascript-3.md [hunk 1]
```diff
@@ -230,6 +230,18 @@ An easy way to achieve this is to add JavaScript code to load the CSS file direc
 
    ![Final result](assets/flatpickr-final.gif)
 
+## Troubleshooting
+If your javascript integration is not behaving properly, you can troubleshoot with hints below.
```
Comment (davidjohnoliver) [inline doc/articles/interop/wasm-javascript-3.md:234]:
```suggestion
If your JavaScript integration is not behaving properly, you can troubleshoot with hints below.
```

File: doc/articles/interop/wasm-javascript-3.md [hunk 2]
```diff
@@ -230,6 +230,18 @@ An easy way to achieve this is to add JavaScript code to load the CSS file direc
...
+
+### My javascript control does not accept pointer input
```
Comment (davidjohnoliver) [inline doc/articles/interop/wasm-javascript-3.md:236]:
```suggestion
### My JavaScript control does not accept pointer input
```

File: doc/articles/platform-specific-csharp.md [hunk 3]
```diff
@@ -32,9 +32,9 @@ The structure of an Uno app created with the default [Visual Studio template](ht
  | UWP         | `NETFX_CORE`  |
  | Android     | `__ANDROID__` |
  | iOS         | `__IOS__`     |
- | WebAssembly | `__WASM__`    |
+ | WebAssembly | `HAS_UNO_WASM`|
```
Comment (davidjohnoliver) [inline doc/articles/platform-specific-csharp.md:35]: Huh! Why?

**Review by davidjohnoliver — APPROVED** [review]

File: doc/articles/platform-specific-csharp.md (hunk 3 above)
Comment (jeromelaban) [reply doc/articles/platform-specific-csharp.md:35]: Both are valid, but I'd rather to use a more uno-specific one (aligned with what the SKIA runtime is exposing).

Comment (ajpinedam) [reply doc/articles/platform-specific-csharp.md:35]: That one is new to me 👍

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**